bpf/lib/conntrack.h

```diff
@@ -254,14 +255,15 @@ static __always_inline __u8 __ct_lookup(const void *map, struct __ctx_buff *ctx,
 			}
 			break;
 		case ACTION_CLOSE:
+		case ACTION_RESET:
```

Comment:
nit: Please add `/* fallthrough */` to document that the cases here are intentional:

```
case ACTION_CLOSE:
    /* fallthrough */
case ACTION_RESET:
```